source/Ocl/OclWriter.cs

```diff
@@ -252,7 +252,7 @@ void WriteValue(OclStringLiteral literal)
             writer.WriteLine();
 
             if (isIndented)
-                WriteIndent(1);
+                WriteIndent(2);
```

Comment:
This ensures the ending tag takes the same 2 indent baseline as the [text block above](https://github.com/OctopusDeploy/Ocl/pull/131/files#diff-b671eafff51692926c4dc6d8f51d1ee70ed08bc2ebf32e0c5f21ce44c3098b74R248)

Reply:
This would make a nice code comment for the next person to come along.

Reply:
+1